instances/measuredQuantity/measuredHoldingPotential.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+      "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/measuredQuantity/measuredHoldingPotential",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/MeasuredQuantity",
+    "definition": "Measured membrane potetial during a voltage-clamp protocol.",
+    "description": "",
+    "interlexIdentifier": null,
+    "knowledgeSpaceLink": null,
+    "name": "measured holding potential",
```

Comment:
why measured? why not only "holding potential"?

Reply:
because we have "targetHoldingPotential" and "measuredHoldingPotential". Maybe not essential to have "measured" in the name here since "measured" is in the name of the controlled term type, but it seemed safest to keep it.

Reply:
I see.